pkg/kubernetes/kubernetes.go

```diff
@@ -70,16 +72,22 @@ func New(s v1alpha1.Spec) (*Kubernetes, error) {
 type ApplyOpts client.ApplyOpts
 
 // Apply receives a state object generated using `Reconcile()` and may apply it to the target system
-func (k *Kubernetes) Apply(state manifest.List, opts ApplyOpts) error {
+func (k *Kubernetes) Apply(baseDir string, state manifest.List, opts ApplyOpts) error {
 	info, err := k.ctl.Info()
 	if err != nil {
 		return err
 	}
 	alert := color.New(color.FgRed, color.Bold).SprintFunc()
 
 	if !opts.AutoApprove {
+		var message string
+		if opts.Prune {
+			message = `Applying to and pruning namespace '%s' of cluster '%s' at '%s' using context '%s'.`
+		} else {
+			message = `Applying to namespace '%s' of cluster '%s' at '%s' using context '%s'.`
```

Comment:
No `if else` dance please, just default to the one without prune and reassign in case

Reply:
huh? Explain? This may be a Golang thing I've not heard.

Reply:
```go
message := `Applying to namespace '%s' of cluster '%s' at '%s' using context '%s'.`
if opts.Prune {
  message = `Applying to and pruning namespace '%s' of cluster '%s' at '%s' using context '%s'.`
}
```